beluga_amcl/cmake/ament.cmake

```diff
@@ -0,0 +1,95 @@
+find_package(ament_cmake REQUIRED)
```

Comment:
Ah, CI is complaining that some of these files are missing copyright notices:

```
  1: cmake/ament.cmake: could not find copyright notice
  1: cmake/beluga_amcl-extras.cmake: could not find copyright notice
  1: cmake/catkin.cmake: could not find copyright notice
  1: cmake/lib/nodelets.cmake: could not find copyright notice
  1: test/cmake/ament.cmake: could not find copyright notice
  1: test/cmake/catkin.cmake: could not find copyright notice
```

Reply:
And indeed they are. Will fix.

Reply:
Fixed in 9afc755ad1b67b1069c0e313270ebbe1e633629a.